rskj-core/src/main/java/org/ethereum/config/Constants.java

```diff
@@ -210,11 +242,16 @@ public static Constants devnetWithFederation(List<BtcECKey> federationPublicKeys
                 new BlockDifficulty(BigInteger.valueOf((long) 14E15)),
                 BigInteger.valueOf(50),
                 540,
-                new BridgeDevNetConstants(federationPublicKeys)
+                new BridgeDevNetConstants(federationPublicKeys),
+                new BlockDifficulty(new BigInteger("550000000"))
         );
     }
 
     public static Constants testnet() {
+        return testnet(null);
```

Comment:
This constructor is used only in tests and looks dangerous to use outside of them (will always return previous difficulty, even when activation has passed). Maybe delete it?